tests/test_repoconf.c

```diff
@@ -519,8 +503,8 @@ START_TEST(test_write_repoconf)
     _cleanup_error_free_ GError *tmp_err = NULL;
 
     // Create a temporary file
-    rc = mkstemp(tmpfn);
-    fail_if(rc == -1);
+    fd = mkstemp(tmpfn);
+    fail_if(fd == -1);
```

Comment:
I know it was like this already, but since `mkstemp` opens the file, maybe we should close it here too. Not that it matters much for the tests, but...

Reply:
Here it is taken care of by `_cleanup_fd_close_` (at line 496), I am honestly not sure why the covscan is ok with it here but wasn't when I tried to use it in the other tests, maybe its because it isn't used here for anything.

Reply:
Oh, I see. Looks good then.